src/include/btmem.h

```diff
@@ -1111,10 +1111,8 @@ struct __wt_update {
  */
 struct __wt_update_value {
     WT_ITEM buf;
-    wt_timestamp_t durable_ts;
-    uint64_t txnid;
+    WT_TIME_WINDOW tw;
```

Comment:
Do we need time window ? Do we need both start and stop time pairs to be associated with __wt_update_value ?

Reply:
Yes. For the history store to retrieve the exact time window, we need to retrieve the entire time window.  For data stores, the start timestamp is used when the update is a standard update and stop time pair when the update is a tombstone.